diff -urN mono-debugger-2.6/configure.in mono-debugger-2.6.new//configure.in
--- mono-debugger-2.6/configure.in	2009-09-25 17:19:09.000000000 +0200
+++ mono-debugger-2.6.new//configure.in	2010-03-11 20:07:54.397473048 +0100
@@ -256,7 +256,7 @@
 AC_SUBST(LIB_SUFFIX)
 
 AC_CHECK_HEADERS(stddef.h string.h strings.h stdlib.h time.h unistd.h)
-AC_CHECK_HEADERS(fcntl.h sys/user.h sys/file.h sys/time.h termcap.h termcap/termcap.h)
+AC_CHECK_HEADERS(fcntl.h sys/user.h sys/file.h sys/time.h ncurses/termcap.h)
 AC_HEADER_TIME
 AC_HEADER_DIRENT
 AC_CHECK_FUNCS(fcntl getpagesize setitimer sysconf fdopen getuid getgid)
